lib/network.sh

```diff
@@ -189,16 +191,16 @@ else
   export REGISTRY=${REGISTRY:-"[${EXTERNAL_SUBNET_V6_HOST}]:${REGISTRY_PORT}"}
 fi
 
-network_address INITIAL_IRONICBRIDGE_IP "$PROVISIONING_NETWORK" 9
+network_address INITIAL_BARE_METAL_PROVISIONER_BRIDGE_IP "$BARE_METAL_PROVISIONER_NETWORK" 9
 
-export DEPLOY_KERNEL_URL=${DEPLOY_KERNEL_URL:-"http://${CLUSTER_URL_HOST}:${HTTP_PORT}/images/ironic-python-agent.kernel"}
-export DEPLOY_RAMDISK_URL=${DEPLOY_RAMDISK_URL:-"http://${CLUSTER_URL_HOST}:${HTTP_PORT}/images/ironic-python-agent.initramfs"}
+export DEPLOY_KERNEL_URL=${DEPLOY_KERNEL_URL:-"http://${CLUSTER_BARE_METAL_PROVISIONER_HOST}:${HTTP_PORT}/images/ironic-python-agent.kernel"}
```

Comment:
Move quotes around the whole assignment. Same for all lines below.

Reply:
done